.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,18 +34,15 @@ See [USWDS release notes](https://github.com/uswds/uswds/releases) for examples.
 -->
 
 ## Breaking change
-_Indicate if this update is a breaking change._
-<!--
-Use clear language to:
-1. Indicate if this update is a breaking change.
-2. Explain what actions are required for the user to remediate the break.
-   For clarity, include a statement here even if there is no breaking change:
-   "This is not a breaking change."
-   ":warning: This is potentially a breaking change."
-   ":warning: This is a breaking change."
--->
+
+_Indicate if this update is a breaking change with one of the following statements:_
+This is not a breaking change.
+:warning: This is potentially a breaking change.
+:warning: This is a breaking change.
```

Comment:
Just an idea, it might be nice to have these as a bulleted list since we're listing out options

> _Indicate if this update is a breaking change with one of the following statements:_
> 
> - This is not a breaking change.
> - :warning: This is potentially a breaking change.
> - :warning: This is a breaking change.

I think the indention also helps with readability

Reply:
A list does help initially, but when updating we'll just end up removing this formatting. I'm inclined to try this out _without_ the list formatting.

That way we can just keep the statement that applies.

Reply:
I agree with @mejiaj on this one. In other circumstances I would advocate for list formatting, but since this content exists only for users to quickly delete two of the three options, I am inclined to say that less formatting (and quicker deleting) is better. We can always adjust if we see that users are getting confused.